src/config/service_factory.py

```diff
@@ -1,9 +1,18 @@
 from azure.servicebus import ServiceBusClient
+from libsuitecrm import SuiteCRM  # type: ignore
 
 from .service_factory_interface import IServiceFactory
 
 
-class AzureServiceFactory(IServiceFactory):
+class ServiceFactory(IServiceFactory):
 
     def get_service_bus_client(self, conn_str: str, enable_logging: bool = False) -> ServiceBusClient:
         return ServiceBusClient.from_connection_string(conn_str, logging_enable=enable_logging)
+
+    def get_suitecrm_client(self) -> SuiteCRM:
+        return SuiteCRM(
+            self._config["DATA_REGISTRY_SUITECRM_API_URL"],
+            self._config["DATA_REGISTRY_SUITECRM_CLIENT_ID"],
+            self._config["DATA_REGISTRY_SUITECRM_CLIENT_SECRET"],
+            secure=self._config["DATA_REGISTRY_SUITECRM_SECURE"] == "true",
```

Comment:
Can we use `!= "false"` here, so that if it's an invalid value it defaults to secure?